content/components/accordion/rationale/usage.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: component/rationale
+heading: When to use accordions
+---
+
+We recommend that accordions be used sparingly for primary content on a page.  While they can be appropriate for organising small, specific sections of content, accordions are not a suitable replacement for plain text that is well formatted. If a user requires all, or most of the information on the page it should be visible, not hidden inside an accordion. 
```

Comment:
Either: "...plain text that is well-formatted" Or: "...well formatted plain text."